Client/Frontend/Browser/BraveWebView.swift

```diff
@@ -1,9 +1,23 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import Foundation
 import WebKit
-import Shared
-import JavaScriptCore
 
-class BraveWebView: TabWebView {    
+class BraveWebView: WKWebView {
+
+    init(frame: CGRect, configuration: WKWebViewConfiguration = WKWebViewConfiguration(), privacyProtection: PrivacyProtectionProtocol = PrivacyProtection()) {
+        if privacyProtection.nonPersistent {
+            configuration.processPool = WKProcessPool()
+            configuration.websiteDataStore = WKWebsiteDataStore.nonPersistent()
+        }
+
+        super.init(frame: frame, configuration: configuration)
+    }
+
+    required init?(coder: NSCoder) {
```

Comment:
I think we should make this unavailable:

`@available(*, unavailable)`

and throw a `fatalError()` if used (this is elsewhere in our code).

Reply:
👍